R_Friedmantest.qmd

```diff
@@ -0,0 +1,52 @@
+---
+title: "Friedman Chi-Square test using R package rstatix"
+---
+
+# **Example: Simulated dataset with 10 subjects each having results at each of the 4 time points.**
+
+Simulated datasets with continuous endpoints are generated for single-drug repeated measurements to check whether any significance exists between the responses at different time points. The p-value will indicate whether differences in response for different time points are significant.
+
+*Data source:*
+
+```{r, output = FALSE, message = FALSE}
+library(dplyr)
+library(rstatix) 
+```
+
+```{r eval=FALSE}
+set.seed(123)
+
+Onewayrepeat = data.frame(subject = rep(1:10, each=4),
```

Comment:
We often try to follow a similar style when using R. While this is a personal preference of course, the tidyverse style guide is a helpful resource. https://style.tidyverse.org/syntax.html#:~:text=Generally%2C%20variable%20names%20should%20be,this%20is%20not%20easy!).&text=Where%20possible%2C%20avoid%20re%2Dusing,the%20readers%20of%20your%20code.
Using this type of styling we would reformat your variable name as below:
```suggestion
one_way_repeat <- data.frame(subject = rep(1:10, each=4),
  timepoint = rep(c(1,2,3,4), times = 10),
  response = round(runif(n=40, 10, 50))
```